### PR TITLE
Domain changes after project being transferred

### PR DESCRIPTION
**Pull request by paualarco**
- Package renaming (erased cloriko)
- Added contributing and code of conduct and authors files.
- Header licenses for all projects, and shared settings on sbt
- Added some todos: (Sonatype, scaladoc, release configs and disabled fatal warning scala compiler flag.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Authors that have contributed code, documentation, tests and feedback
+to project Monix Connect and that own the copyright of their contributions:
+
+Alexandru Nedelcu
+https://github.com/alexandru
```
Comment (alexandru) [inline AUTHORS:5]: I don't mind being here, but we aren't authors yet 🙂

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+We are committed to providing a friendly, safe and welcoming environment for all, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other such characteristics.
+
+Everyone is expected to follow the [Scala Code of Conduct] when discussing the project on the available communication channels. If you are being harassed, please contact us immediately so that we can support you.
+
+## Moderation
+
+Any questions, concerns, or moderation requests please contact a member of the project.
+
+- Alexandru Nedelcu: [gitter](https://gitter.im/alexandru) | [twitter](https://twitter.com/alexelcu) | [email](mailto:coc@temp18.alexn.org)
```
Comment (alexandru) [inline CODE_OF_CONDUCT.md:11]: Oh, this reminds me to update my email address, good thing I took a look. Please update to:

```
y20+coc@alexn.org
```

In public I like publishing dynamic email addresses and to change them, to combat spam. Unfortunately I have to keep track of where my address ends up at for when I change it 🙂

File: build.sbt [hunk 3]
```diff
@@ -1,90 +1,209 @@
...
+            if e.label == "dependency" && e.child.exists(child =>
+              child.label == "groupId" && child.text == "org.scoverage") =>
+          Nil
+        case _ => Seq(node)
+      }
+    }).transform(node).head
+  },
+  licenses      := Seq("APL2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
+  //homepage := Some(url("https://monix.io")), //todo homepage settings
+  headerLicense := Some(HeaderLicense.Custom("""|Copyright (c) 2014-2020 by The Monix Project Developers.
```
Comment (alexandru) [inline build.sbt:111]: I have this preference, nay obsession, for indenting to not look like this with things hanging from the far right. It looks like a branch of some sort 🙂

Just my preference, no need to do anything about it.

File: common/src/main/scala/monix/connect/common/Operators.scala [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
```
Comment (alexandru) [inline common/src/main/scala/monix/connect/common/Operators.scala:2]: You might want to say "Monix Connect". And link to the project's repository, until we'll settle for a documentation website.

As I said, we could do some arrangement with `monix.io`, maybe have a `monix.io/connect/` or a `connect.monix.io`, but until then let's link to the repository, such that the content is related to Connect.

File: common/src/main/scala/monix/connect/common/Operators.scala [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+import monix.reactive.Observable
+
+object Operators {
+
+  type Transformer[A, B] = Observable[A] => Observable[B]
+
+  object Implicits {
+    implicit class ObservableExtension[A](ob: Observable[A]) {
+      def transform[B](transformer: Transformer[A, B]): Observable[B] = {
```
Comment (alexandru) [inline common/src/main/scala/monix/connect/common/Operators.scala:28]: Hmm, maybe we should have kept this in Monix 🙂

Comment (Avasil) [reply common/src/main/scala/monix/connect/common/Operators.scala:28]: There is some demand: https://github.com/monix/monix/issues/891

**Pull request merged**